api/client/service/update.go

```diff
@@ -122,8 +117,8 @@ func updateService(spec *swarm.ServiceSpec, flags *pflag.FlagSet) error {
 	updateString(flagName, &spec.Name)
 	updateLabels(flags, &spec.Labels)
 	updateString("image", &cspec.Image)
-	updateSlice("command", &cspec.Command)
-	updateSlice("arg", &cspec.Command)
+	updateStringToSlice(flags, "command", &cspec.Command)
```

Comment:
I based that statement off of what [I see in swarmkit](https://github.com/docker/swarmkit/blob/b4644d8/agent/exec/container/container.go#L95-L108). I'm not seeing any command + args.

The cli just passes through what the user sets for those values.

The cli does